src/css/footer.css

```diff
@@ -0,0 +1,299 @@
+.footer-background {
+  background-color: var(--dark);
+}
+
+.footer-container {
+  padding-top: 40px;
+  padding-bottom: 10px;
+}
+
+@media screen and (min-width: 768px) {
+  .footer-container {
+    padding-top: 64px;
+  }
+}
+
+@media screen and (min-width: 768px) {
+  .footer-inner-container {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+  }
+}
+
+.footer-top-wrapper {
+  display: flex;
+  flex-direction: column;
+  gap: 20px;
+}
+
+@media screen and (min-width: 768px) {
+  .footer-top-wrapper {
+    gap: 32px;
+  }
+}
+
+@media screen and (min-width: 1440px) {
+  .footer-bottom-wrapper {
+    display: flex;
+    flex-direction: row;
+    gap: 115px;
+  }
+}
+
+.footer-logo {
+  fill: var(--light);
+}
+
+.footer-socials-list {
+  display: flex;
+  gap: 12px;
+}
+
+@media screen and (min-width: 768px) {
+  .footer-socials-list {
+  }
+}
+
+.socials-link {
```

Comment:
We already have social links on the black background in the mobile menu. Please move those styles to common place, or use the same markup. Thus you can remove your own css styles

Reply:
Used styles from the header, but still needed to leave footer styles due to different logic with adaptive versions